thinc/layers/tensorflowwrapper.py

```diff
@@ -1,10 +1,19 @@
-from typing import Any, Callable, Optional, Tuple, Type, TypeVar
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type, TypeVar
 
+import numpy as np
+
+from ..config import registry
 from ..model import Model
 from ..shims import TensorFlowShim
-from ..util import xp2tensorflow, tensorflow2xp, assert_tensorflow_installed
-from ..util import is_tensorflow_array, convert_recursive, is_xp_array
-from ..types import ArgsKwargs
+from ..types import ArgsKwargs, Array
+from ..util import (
```

Comment:
Nitpick, but we try to avoid this by splitting the imports over multiple lines and repeating the `from` etc. It's not ideal either but it's less verbose.

Reply:
Sorry, I use the "organize imports" functionality in VSCode and then run black on top of it. I'll check out isort

Reply:
Just tested isort and it's pretty cool, but there's no way to make it work without wrapping imports like this 😞 There's an open issue on the tracker about a mode that preserves multiple lines importing from the same source (which black leaves untouched, too), but it's not supported yet. So I'm not sure if we want to adpot it... I ran it over the whole code base and it does make things a lot more verbose.

Reply:
I'll keep an eye out for import wrapping and break the lines manually for now

Reply:
I'll keep an eye out for import wrapping and break the lines manually for now